cegs_portal/search/views/custom_views.py

```diff
@@ -49,8 +77,13 @@ def get_experiment_accession_id(self):
         )
 
 
+def default_tsv_renderer(data):
+    raise NotImplementedError("FeatureEffectsView.get_data")
```

Comment:
Oh, we should change the message here. Maybe something like this:
```suggestion
    raise NotImplementedError("No TSV output for this endpoint")
```

I think we should have something similar for the default json renderer (in another PR)